non-wheel/httpcrtsrv/httpsrv.py

```diff
@@ -0,0 +1,444 @@
+        print('conf: %s' % conf)
+        print('name: %s' % name)
+        print('port: %s' % port)
+        os.mkdir('%s' % path)
+        os.mkdir('%s/conf' % path)
+        os.mkdir('%s/logs' % path)
+        os.mkdir('%s/htdocs' % path)
+        index_file = open("%s/htdocs/index.html" % path, "w+")
+        index_file.write(get_index_html(name))
+
+        conf_file = open("%s/conf/httpd.conf" % path, "w+")
+        fastcgi_conf_file = open("%s/conf/fastcgi.conf" % path, "w+")
+        fastcgi_conf_twn_file = open("%s/conf/fastcgi.conf.twn" % path, "w+")
+        fastcgi_dynamic_conf_file = open("%s/conf/fastcgi_dynamic.conf" % path, "w+")
+        fastcgi_http_add_conf_file = open("%s/conf/fastcgi_http_add.conf" % path, "w+")
+        fastcgi_conf_file.write(get_fastcgi_conf(name))
+        fastcgi_conf_twn_file.write(get_fastcgi_conf_twn(name))
+        fastcgi_dynamic_conf_file.write(get_fastcgi_dynamic_conf(name))
+        fastcgi_http_add_conf_file.write(get_fastcgi_http_add_conf(name))
+        fastcgi_conf_file.close()
+        fastcgi_conf_twn_file.close()
+        fastcgi_dynamic_conf_file.close()
+        fastcgi_http_add_conf_file.close()
+
+        if conf == "apache":
+            conf_file.write(get_apache_conf(name, port))
+        elif conf == "zendphp7":
+            conf_file.write(get_zendphp7_conf(name, port))
+        elif conf == "zendsvr6":
+            conf_file.write(get_zendsvr6_conf(name, port))
+
+        conf_file.close()
+
+        os.mkdir('%s/conf/apache-sites' % path)
+        os.mkdir('%s/example-project' % path)
+        os.mkdir('%s/example-project/conf' % path)
+        vhost_file = open("%s/example-project/conf/%s.conf" % (path, name), "w+")
+        vhost_file.write(get_example_vhost_conf(name, port))
+        vhost_file.close()
+        os.mkdir('%s/example-project/htdocs' % path)
+        index_file = open("%s/example-project/htdocs/index.html" % path, "w+")
+        index_file.write(get_index_html('Example Project'))
+        index_file.close()
+        os.mkdir('%s/example-project/logs' % path)
+        os.symlink("%s/example-project/conf/%s.conf" % (path, name), "%s/conf/apache-sites/example-project.conf" % path)
+
+        create_with_sql(name)
+
+
+# Hopefully to go away in favor of create_with_qzui_api
+def create_with_sql(name):
+    # This connection is for the SQL way of creating an HTTP Server Instance.
+    conn = db2.connect()
+    cur = conn.cursor()
+
+    sys_cmd = "CPYF FROMFILE(QUSRSYS/QATMHINSTC) TOFILE(QUSRSYS/QATMHINSTC) " +\
+            "FROMMBR(APACHEDFT) TOMBR(%s) MBROPT(*REPLACE)" % name
+    crt_http_svr = "call qcmdexc('%s');" % sys_cmd
+    cur.execute(crt_http_svr)
+
+    # This is the part that does not work properly.
+    mod_info_query = '''
+create or replace alias QUSRSYS.QATMHINSTC_%s for QUSRSYS.QATMHINSTC(%s);
+update QUSRSYS.QATMHINSTC_%s
+set CHARFIELD = '-apache -d /www/%s -f conf/httpd.conf -AutoStartN';
+''' % (name, name, name, name)

```

Comment:
You can't execute multiple queries at once in `ibm_db_dbi`. You will have to split these in to two queries:

```python
mod_info_query = 'create or replace alias QUSRSYS.QATMHINSTC_{0} for QUSRSYS.QATMHINSTC({0})'.format(name)
cur.execute(mod_info_query)

mod_info_query = "update QUSRSYS.QATMHINSTC_{0} set CHARFIELD = '-apache -d /www/{0} -f conf/httpd.conf -AutoStartN'".format(name)
cur.execute(mod_info_query)
```